cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -133,15 +133,11 @@
 
                     {% else %}
 
-                    <span class="{{ _classes( nav_depth, '-overview-heading' ) }}
-                               {{ _classes( nav_depth, '-overview-heading-link' ) }}
-                    ">
-                        <a class="{{ _classes( nav_depth, '-overview-link' ) }}
-                              {{ _classes( nav_depth, '-overview-link__current' ) if nav_overview_url == request.path else '' }}"
-                        {{ '' if nav_overview_url == request.path or nav_overview_url == '#' else 'href=' + nav_overview_url | e }}>
-                        {{ nav_overview_text ~ ' Overview' }}
-                        </a>
-                    </span>
+                    <a class="{{ _classes( nav_depth, '-overview-link' ) }}
+                          {{ _classes( nav_depth, '-overview-link__current' ) if nav_overview_url == request.path else '' }}"
```

Comment:
Minor, indentation should be aligned for the classes and the string should be four spaces.